week-2-wide-perspective/w02e05-simple-server-side/src/auth/auth.service.ts

```diff
@@ -0,0 +1,14 @@
+import { SignJWT, importPKCS8, jwtVerify } from 'jose'
+
+const getPrivateKey = async () => importPKCS8(process.env.PRIVATE_KEY ?? '', 'ES256')
```

Comment:
Fajnie jakby to spakować w jakiś obiekt "namespace" w sensie:
```typescript
export const authService = {
   async getPrivateKey() {...},
   async createSignedJWT(...) {...},
}
```

etc.

Ponieważ jak Ci urośnie system - to wtedy będzie ciężko się zorientować (po drugiej stronie - tam gdzie `import`) co jest z czym związane.